src/misc/lv_anim.c

```diff
@@ -274,45 +274,46 @@ int32_t lv_anim_path_overshoot(const lv_anim_t * a)
 int32_t lv_anim_path_bounce(const lv_anim_t * a)
 {
     /*Calculate the current step*/
-    int32_t t = lv_map(a->act_time, 0, a->duration, 0, LV_BEZIER_VAL_MAX);
+    int32_t t = lv_map(a->act_time, 0, a->duration, 0, 1024);
```

Comment:
Why change this `LV_BEZIER_VAL_MAX` -> 1024?

Reply:
Thanks, fixed!